### PR TITLE
Files docstrings

### PR DESCRIPTION
**Pull request by tomvanmele**
### What type of change is this?

- [ ] Bug fix in a **backwards-compatible** manner.
- [ ] New feature in a **backwards-compatible** manner.
- [ ] Breaking change: bug fix or new feature that involve incompatible API changes.
- [x] Other (e.g. doc update, configuration, etc)

### Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I added a line to the `CHANGELOG.md` file in the `Unreleased` section under the most fitting heading (e.g. `Added`, `Changed`, `Removed`).
- [ ] I ran all tests on my computer and it's all green (i.e. `invoke test`).
- [ ] I ran lint on my computer and there are no errors (i.e. `invoke lint`).
- [ ] I added new functions/classes and made them available on a second-level import, e.g. `compas.datastructures.Mesh`.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: src/compas/files/ply.py [hunk 1]
```diff
@@ -7,20 +7,27 @@
...
-    'PLY',
-    'PLYReader',
-    'PLYParser',
-    'PLYWriter',
-]
-
 
 class PLY(object):
-    """Polygon file format, or Stanford triangle format.
+    """Class for working with files in Polygon format, or Stanford triangle format.
```
Comment (gonzalocasas) [inline src/compas/files/ply.py]:
```suggestion
    """Class for working with files in Polygon format, also known as Stanford triangle format.
```

File: src/compas/files/urdf.py [hunk 2]
```diff
@@ -204,8 +200,17 @@ def install_parser(cls, parser_type, *tags):
...
             One or more URDF string tag that the parser can parse.
+
+        Returns
+        -------
+        None
+
+        Raises
+        ------
+        ValueError
+            If `tags` is empty.
```
Comment (gonzalocasas) [inline src/compas/files/urdf.py:213]:
```suggestion
            If ``tags`` is empty.
```

**Review by gonzalocasas — APPROVED** [review]
LGTM!

File: src/compas/files/urdf.py (hunk 2 above)
Comment (tomvanmele) [reply src/compas/files/urdf.py:213]: i got a bit confused with these guys.

if i understand correctly, variable names should be \`x\`. only values and code snippets should have \`\`x = 0\`\`

**Pull request merged**